documentation/modules/rn-2.6.adoc

```diff
@@ -317,3 +297,106 @@ In earlier releases of {project-short}, an earlier failed migration could have l
 
 
 For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12436210[Resolved Issues] in Jira.
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+
+.Disks and drives are offline after migrating Windows virtual machines from RHV or VMware to OCP
```

Comment:
This bug is actually fixed in 2.6.4!

But certain disks (windows "dynamic disks") are still impacted, see https://issues.redhat.com/browse/MTV-1344